### PR TITLE
Fix integration tests.

### PR DESCRIPTION
**Pull request by jseabold**
## Todos
- [x] MIT compatible
- [x] Tests

Fixing test as suggested in #91.

### CODE REVIEW

File: tests/test_postfetch_lastrowid.py [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 def test_postfetch_lastrowid(redshift_session):
     session = redshift_session
-    for i in range(5):
-        postfetchExample = models.PostfetchExample(some_int=i)
-        session.add(postfetchExample)
+    postfetch_examples = [models.PostfetchExample(some_int=i) for i in range(5)]
```
Comment (graingert) [inline tests/test_postfetch_lastrowid.py]: rename this variable "examples"

**Comment by jseabold** [comment]
Squashed.

**Comment by jseabold** [comment]
Should I just push this branch to the main org repo and re-open a PR?

**Comment by graingert** [comment]
Looks like this fails sometimes. On master it returned all zeroes. Is this a redshift bug?

**Pull request merged**